deploy.rb

```diff
@@ -47,16 +51,26 @@ def make_deploymentdir
     make_dir(DEPLOY_DIR)
   end
 
-  def clone_versions
-    @versions.each { |version| clone_version(version) }
+  def clone_versions(remote)
+    @versions.each { |version| clone_version(version, remote) }
   end
 
-  def clone_version(version)
+  def clone_version(version, remote)
+    if remote == 'origin'
+      remote_url = 'git://github.com/Katello/katello.org' 
+    else   
+      remote_url = syscall("git remote  show jlsherri | grep Fetch | awk '{print $3}'")
```

Comment:
Or maybe `git ls-remote --get-url #{remote}` is better, but anything with out jlsherri :smile: